catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -113,9 +113,9 @@ const SearchForm = ({
                   action: 'clear search',
                   label: 'works-search'
                 });
-                const link = worksUrl({query: null, page: null});
+
+                // TODO: re-focus input (useRef?)
                 setQuery('');
-                Router.push(link.href, link.as);
```

Comment:
How is the route changed?

Reply:
It isn't – when we clear out the search box by clicking this button, we don't want to change the route, we just want an empty input. Currently, you go back to the /works landing page, which feels broken.

Reply:
Ah, sorry, my bad, didn't realise it was on the `ClearSearch`.